python/lsst/meas/algorithms/ingestIndexReferenceTask.py

```diff
@@ -32,6 +35,9 @@
 from lsst.afw.image import fluxFromABMag, fluxErrFromABMagErr
 from .indexerRegistry import IndexerRegistry
 from .readTextCatalogTask import ReadTextCatalogTask
+from .loadReferenceObjects import LoadReferenceObjectsTask
+
+_RAD_PER_DEG = math.pi / 180
```

Comment:
Two reasons:
- I prefer to avoid numpy for scalars, for fear of inefficiencies and because it tends to confuse readers (who may think it is operating on array-like data)
- I personally dislike `math.radians` and `np.radians` because I feel their meaning is opaque.